frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetToRemoteAction.java

```diff
@@ -148,7 +148,7 @@ public void actionPerformed(AnActionEvent anActionEvent) {
     }
 
     // if key is missing the default value (false) is returned
-    if (!PropertiesComponent.getInstance().getBoolean(RESET_INFO_SHOWN)) {
+    if (!PropertiesComponent.getInstance().getBoolean(DONT_SHOW_RESET_INFO)) {
```

Comment:
Actually AFAICS this is a double negation... it'd be clearer to use non-negated key, `SHOW_RESET_INFO`. The problem would be then that the default would need to be `true`... is there some `getBooleanOrDefault(..., ...)` method available?